draft-ietf-quic-http.md

```diff
@@ -402,15 +431,25 @@ without EHB set, followed by a HEADERS frame with EHB set.
 ### PRIORITY {#frame-priority}
 
 The PRIORITY (type=0x02) frame specifies the sender-advised priority of a stream
-and is substantially different from {{!RFC7540}}. In order to support ordering,
-it MUST be sent only on the control stream. The format has been modified to
-accommodate not being sent on-stream and the larger stream ID space of QUIC.
+and is substantially different in format from {{!RFC7540}}.  In order to ensure
```

Comment:
This is an echo of language elsewhere in the HTTP draft, indicating which frames are mostly-direct copies of the HTTP/2 equivalent, and which ones are fundamentally different.  Happy to discuss rephrasing that all-up in a separate editorial issue, but I think following the existing pattern in this PR is entirely appropriate.